app/LVM/README.md

```diff
@@ -0,0 +1,463 @@
+
+The size of a linear LVM logical volume (LV).
+
+##### Trigger Prototypes
+* LVM linear LV {#LVM_LV_FULL_NAME} is not active ({ITEM.VALUE1}) on {HOSTNAME} (`{Template App LVM active:rabe.lvm.lvs.value[{#LVM_LV_FULL_NAME},lv_active,linear].str(active)}=0`)
+
+The linear LVM logical volume is not in state "active", which is most likely a sign of a problem.
+#### LVM logical thick snapshot volumes discovery (`rabe.lvm.lvs.discovery[thicksnapshot]`)
+
+Low-Level discovery of a thick provisioned snapshot LVM logical volumes (LV)
+
+Returns the following macros for each available LV:
+* `{#LVM_LV_FULL_NAME}`
+  * Logical volume full name (`lvs -o lv_full_name`)
+  * Example: `vg_example_01/lv_example_01`
+* `{#LVM_LV_UUID}`
+  * Logical volume UUID (`lvs -o lv_uuid`)
+  * Example. `dxi5Hw-Xo2m-SAVi-ypZh-QO4I-krpI-sR3N67`
+* `{#LVM_LV_TYPE}`
+  * Logical volume segment type (`lvs -o segtype`)
+  * Example: `linear`, `thin`, `thin-pool`
+* `{#LVM_LV_ROLE}`
+  * Logical volume role (`lvs -o lv_role`)
+  * Example: `public`, `private`, `public,snapshot,thicksnapshot`
+* `{#LVM_LV_NAME}`
+  * Logical volume name (`lvs -o lv_name`)
+  * Example: `lv_example_01`
+* `{#LVM_LV_VG_NAME}`
+  * Volume group name (`lvs -o vg_name`)
+  * Example: `vg_example_01`
+* `{#LVM_LV_PATH}`
+  * Full pathname for LV. Blank for internal LVs. (`lvs -o lv_path`)
+  * Example: `/dev/vg_example_01/lv_example_01`
+
+##### Item Prototypes
+* Active state of $3 LV $1 (`rabe.lvm.lvs.value[{#LVM_LV_FULL_NAME},lv_active,thicksnapshot]`)
+
+Active state of a thick provisioned LVM snapshot.
+
+* Size of $3 LV $1 (`rabe.lvm.lvs.value[{#LVM_LV_FULL_NAME},lv_size,thicksnapshot]`)
+
+The size of a thick provisioned snapshot LVM logical volume (LV).
+
+* Origin of $3 LV $1 (`rabe.lvm.lvs.value[{#LVM_LV_FULL_NAME},origin,thicksnapshot]`)
+
+Origin device of a thick provisioned LVM snapshot.
+
+* Free snapshot data of $3 LV $1 (`rabe.lvm.lvs.value[{#LVM_LV_FULL_NAME},snap_free,thicksnapshot]`)
+
+The free snapshot data of a thick provisioned LVM snapshot in bytes.
+
+* Free snapshot data of $3 LV $1 in % (`rabe.lvm.lvs.value[{#LVM_LV_FULL_NAME},snap_free_percent,thicksnapshot]`)
+
+The percentage of free snapshot data of a thick provisioned LVM snapshot.
+
+* Allocated snapshot data of $3 LV $1 in % (`rabe.lvm.lvs.value[{#LVM_LV_FULL_NAME},snap_percent,thicksnapshot]`)
+
+The percentage of allocated snapshot data of a thick provisioned LVM snapshot in bytes.
+
+* Allocated snapshot data of $3 LV $1 (`rabe.lvm.lvs.value[{#LVM_LV_FULL_NAME},snap_used,thicksnapshot]`)
+
+The allocated snapshot data of a thick provisioned LVM snapshot in bytes.
+
+##### Trigger Prototypes
+* LVM thick snapshot allocation {HOST.NAME}:[{#LVM_LV_FULL_NAME} is high (< $2 %, {ITEM.VALUE1} free) (`{Template App LVM active:rabe.lvm.lvs.value[{#LVM_LV_FULL_NAME},snap_free,thicksnapshot].last()} < {$LVM_LV_SNAPSHOT_DATA_FREE_WARNING_THRESHOLD:"{#LVM_LV_FULL_NAME}"}

```

Comment:
XSLT needs fixing: Trigger expression can be very long and span multiple lines, it should be added into an indented code block (three code back ticks), rather than a inline code.